packages/melos/lib/src/commands/analyze.dart

```diff
@@ -110,10 +110,8 @@ mixin _AnalyzeMixin on _Melos {
   }) {
     final options = _getOptionsArgs(fatalInfos, fatalWarnings, concurrency);
     return <String>[
-      ..._analyzeCommandExecArgs(
-        useFlutter: workspace.isFlutterWorkspace,
-        workspace: workspace,
-      ),
+      workspace.sdkTool('dart'),
+      'analyze',
```

Comment:
I'm not that familiar with tools like fvm, will they override `dart` too if they are added with `sdkPath`?
Otherwise I agree that it should be enough to just use dart here.

Reply:
Yep, the `fvm ` will use the Dart version that comes with the corresponding Flutter version!